.github/workflows/build-image.yml

```diff
@@ -18,23 +18,28 @@ on:
       IMAGE_REPO:
         required: true
         type: string
-      TAG:
+      BUILD_TYPE:
         required: true
         type: string
```

Comment:
```suggestion
```

Is this input used anywhere in this workflow? It doesn't appear to be. If not, it can be removed.